src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/methodlength/Example1.java

```diff
@@ -0,0 +1,32 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="MethodLength"/>
+  </module>
+</module>
+*/
```

Comment:
I am fine to make an exception here and not use the default at all in the documentation. No user wants to read through 150 lines of code just to get the idea :) Let's just make this max 4 to show how the check works.

Reply:
done